src/Microsoft.DotNet.Wpf/src/System.Xaml/System/Xaml/MS/Impl/KnownStrings.cs

```diff
@@ -62,6 +62,11 @@ public static bool Eq(string a, string b)
             return string.Equals(a, b, StringComparison.Ordinal);
         }
 
+        public static bool Eq(ReadOnlySpan<char> a, ReadOnlySpan<char> b)
```

Comment:
Should we add the `in` key word?

```suggestion
        public static bool Eq(in ReadOnlySpan<char> a, in ReadOnlySpan<char> b)
```

Reply:
No. Why?

Reply:
@stephentoub  Sorry. I read some documents and found that it is not necessary to add the `in` key word.